src/Indexing/Indexing.jsx

```diff
@@ -266,9 +268,11 @@ class Indexing extends React.Component {
         thisPointer.retrieveJobOutput(uid).then(
           (resp) => {
             if (resp.data && resp.data.output) {
-              const logsLink = resp.data.output.split(' ')[0];
+              const manifestLink = resp.data.output.split(' ')[0];
+              const logsLink = resp.data.output.split(' ')[1];
```

Comment:
i'm a bit of paranoid here. could it be possible that the sower job only returns 1 link? so this `[1]` would go out of bounds?

Reply:
Yes, this is the current situation in qa-dcp. The sower job is only returning one link. But this code works correctly ~